src/drivers/ll40ls/ll40ls.cpp

```diff
@@ -135,37 +135,37 @@ void start(enum LL40LS_BUS busid, uint8_t rotation)
 			instance = new LidarLiteI2C(bus_options[i].busnum, bus_options[i].devname, rotation);
 
 			if (!instance) {
-				warnx("Failed to instantiate LidarLiteI2C");
+				PX4_WARN("Failed to instantiate LidarLiteI2C");
 				return;
 			}
 
 			if (instance->init() == PX4_OK) {
 				break;
 			}
 
-			warnx("failed to initialize LidarLiteI2C on busnum=%u", bus_options[i].busnum);
+			PX4_WARN("failed to initialize LidarLiteI2C on busnum=%u", bus_options[i].busnum);
 			delete instance;
 			instance = nullptr;
 		}
 	}
 
 	if (!instance) {
-		warnx("No LidarLite found");
+		PX4_WARN("No LidarLite found");
 		return;
 	}
 
-	fd = open(instance->get_dev_name(), O_RDONLY);
+	fd = px4_open(instance->get_dev_name(), O_RDONLY);
 
 	if (fd == -1) {
-		warnx("Error opening fd");
+		PX4_WARN("Error opening fd");
```

Comment:
Why a warning and not an error (others too)?

Reply:
No reason in particular, these were done in separate passes by different people. I'll give it another look.

Reply:
Fixed.